<ui version="4.0" >
 <class>UiPreferences</class>
 <widget class="QDialog" name="UiPreferences" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>590</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>590</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Preference</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../resource/dboxfe.qrc" >
    <normaloff>:/pics/images/preferences.png</normaloff>:/pics/images/preferences.png</iconset>
  </property>
  <layout class="QGridLayout" name="glPreferences" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidgetPreference" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBoxDosbox" >
         <property name="title" >
          <string>DOSBox</string>
         </property>
         <layout class="QGridLayout" name="_2" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <layout class="QVBoxLayout" name="vlDosbox" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="hlDosboxBinary" >
              <property name="spacing" >
               <number>6</number>
              </property>
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="labelDosboxBinary" >
                <property name="minimumSize" >
                 <size>
                  <width>110</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>110</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text" >
                 <string>Binary:</string>
                </property>
                <property name="buddy" >
                 <cstring>lineEditDosboxBinary</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditDosboxBinary" >
                <property name="text" >
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnDosboxBinary" >
                <property name="minimumSize" >
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="sizeIncrement" >
                 <size>
                  <width>24</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="text" >
                 <string/>
                </property>
                <property name="icon" >
                 <iconset resource="../../resource/dboxfe.qrc" >
                  <normaloff>:/pics/images/folder-open_16.png</normaloff>:/pics/images/folder-open_16.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditDosboxVersion" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="alignment" >
               <set>Qt::AlignHCenter</set>
              </property>
              <property name="readOnly" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QGroupBox" name="groupBoxOption" >
         <property name="title" >
          <string>Option</string>
         </property>
         <layout class="QGridLayout" name="glOption" >
          <item row="0" column="0" >
           <widget class="QCheckBox" name="checkBoxWindowsHide" >
            <property name="text" >
             <string>Hide window if dosbox start</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QCheckBox" name="checkBoxStartKeyMapper" >
            <property name="text" >
             <string>Start key mapper</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk" >
       <property name="minimumSize" >
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Ok</string>
       </property>
       <property name="icon" >
        <iconset resource="../../resource/dboxfe.qrc" >
         <normaloff>:/pics/images/apply_16.png</normaloff>:/pics/images/apply_16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel" >
       <property name="minimumSize" >
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="icon" >
        <iconset resource="../../resource/dboxfe.qrc" >
         <normaloff>:/pics/images/cancel_16.png</normaloff>:/pics/images/cancel_16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource/dboxfe.qrc" />
  <include location="../../../dboxfe_old/res/dboxfe.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>UiPreferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>560</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel" >
     <x>558</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
